specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/examples/text/SuccessfulAbstractiveSummarizationTaskResult.json

```diff
@@ -0,0 +1,56 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "{api-version}",
```

Comment:
There's quite a few files that don't have the actual `api-version` in them. IIRC, you shouldn't use any variables in these examples. I also see a lot of "{API key}" and "{Endpoint}".

Reply:
What values should we set for {API key} and {Endpoint} in examples?